docs/Advanced-Usage/Workloads/index.rst

```diff
@@ -1,16 +1,34 @@
-Workloads
+.. _defining-custom-workloads:
+
+Workload Generation (FireMarshal)
 ================
```

Comment:
This is too short for teh subtitle.

Reply:
fixed, it seemed to work anyway which is weird.